src/features/Chat/components/ActiveWindow/Header/index.tsx

```diff
@@ -37,47 +49,68 @@ const Header = ({ chat }: Props) => {
   };
 
   return (
-    <Container>
-      <ProfileInfo>
-        {icons[chat.status]}
-        <MentorName variant="h2">{chat.displayName}</MentorName>
-        {mentor && <MentorBio variant="p">{mentor.statusMessage}</MentorBio>}
-      </ProfileInfo>
-      {isSearchShown ? (
-        <Search hideSearch={hideSearch} />
-      ) : (
-        <Buttons chat={chat} showSearch={showSearch} />
+    <Container tablet={isTablet}>
+      {isTablet && (
+        <IconButton variant="back" sizeInPx={40} onClick={returnToTabletMenu} />
       )}
+      <IconContainer>{icons[chat.status]}</IconContainer>
+      <MentorName variant="h2">{mentor?.name}</MentorName>
+      <MentorBio isTablet={isTablet} variant="p">
+        {mentor?.statusMessage}
+      </MentorBio>
+      <ButtonsWrapper>
+        <Buttons chat={chat} tabletMode={isTablet} />
+      </ButtonsWrapper>
     </Container>
   );
 };
 
-const Container = styled.div`
+const Container = styled.div<{ tablet: boolean }>`
+  align-items: center;
   border-bottom: 1px solid ${palette.greyLight};
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.03);
   box-sizing: border-box;
   display: flex;
   gap: 30px;
-  height: 80px;
-  justify-content: space-between;
-  min-width: 800px;
+  height: ${({ tablet }) => (tablet ? HIGH_ROW_HEIGHT : ROW_HEIGHT)};
+  justify-content: flex-start;
   padding: 14px 40px;
+  width: ${({ tablet }) =>
+    tablet
+      ? '100vw'
+      : `calc(${CONTENT_WIDTH}-${CHAT_MENU_WIDTH}-${CHAT_GAP_WIDTH}})`};
 `;
 
-const ProfileInfo = styled.div`
-  align-items: center;
-  display: flex;
+const IconContainer = styled.div`
+  flex-shrink: 0;
 `;
 
 const MentorName = styled(Text)`
-  display: block;
-  padding-left: 20px;
-  padding-right: 30px;
+  margin: 0;
+  max-width: calc(
+    (
+      ${CONTENT_WIDTH}-${CHAT_MENU_WIDTH}-${CHAT_GAP_WIDTH}-
+        (2 * 40px + 49px + 20px + 2 * 30px + 500px + 30px)
```

Comment:
What is this magic number `40px + 49px + 20px + 2 * 30px + 500px + 30px` also is repeated later. Could we extract it to a variable?

Reply:
Actually these weren't really necessary so I deleted them!